### PR TITLE
Make message name configurable

### PR DESCRIPTION
**Pull request by ikbalkaya**
This PR adds a new optional configuration property to set message name with interpolation support.

Closes https://github.com/ably/kafka-connect-ably/issues/46

### CODE REVIEW

**Review by lmars — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -168,6 +168,8 @@ Configurations that are supported:
   * For this configuration individual channel parameters can be configured. For example, if you define a `client.channel.params` 
 value with `key1=value1_#{topic},key2=value2_#{key}` in your configuration,
 and publish a message to "topic1" with key "key1", the `client.channel.params`  will be configured with `key1=value1_topic1,key2=value2_key1`
+*`message.name`
```
Comment (QuintinWillison) [inline README.md]: For consistency with surrounding bullets:

```suggestion
* `message.name`
```

File: README.md [hunk 2]
```diff
@@ -168,6 +168,8 @@ Configurations that are supported:
...
+  * For example, if you define a `message.name` value with `message_#{topic}_#{key}` in your configuration, and publish a message to "topic1" with key "key1", the `message.name` will be configured with `message_topic1_key1`
```
Comment (QuintinWillison) [inline README.md:173]: This line should have a blank line beneath it, to vertically space it from the following H2.

**Review by QuintinWillison — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -126,7 +126,8 @@ The advanced properties that can be configured for the connector are:
 
 | Property | Description | Type | Default |
 | -------- | ----------- | ---- | ------- |
-| client.async.http.threadpool.size | The size of the asyncHttp threadpool. | *Integer* | 64 |
+| message.name | Ably message name to publish. | *String* |  |
+| client.async.http.threadpool.size | The size of the asyncHttp threadpool. | *Integer* |  |
```
[the remark this answers is not in the document]
Comment (ikbalkaya) [reply README.md]: Oops, that was accidental. Fix :
https://github.com/ably/kafka-connect-ably/pull/48/commits/0570d0c334783292c05e4b78e965c4852ccb807c

File: README.md (hunk 1 above)
Comment (ikbalkaya) [reply README.md]: Funnily, this looks fine on my local machine, I will commit your suggestion here neverthless.

Comment (ikbalkaya) [reply README.md]: Oops, I'm getting "outdated suggestions cannot be applied"

File: README.md (hunk 2 above)
Comment (ikbalkaya) [reply README.md:171]: https://github.com/ably/kafka-connect-ably/pull/48/commits/147fc97609428df73b79724595674a020bcb14bc

**Review by KacperKluka — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (QuintinWillison) [reply README.md]: Looks like it's fixed in current state of this branch. 🤷 
https://raw.githubusercontent.com/ably/kafka-connect-ably/make_message_name_configurable/README.md

**Review by QuintinWillison — APPROVED** [review]

**Pull request merged**